libs/components/layout/src/lib/modules/box/box-header.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="sky-box-header">
-  <span skyTrim><ng-content /></span
+  <span id="sky-sr-label-box" skyTrim><ng-content /></span
```

Comment:
This needs a dynamically-generated ID instead of a hard-coded ID.

Reply:
I explained this in the description and in stand up--I am going to remove all of the box changes and do it properly in a different PR. I left them in so that the reviewer could do manual testing if they wished, because otherwise you couldn't actually see the changes anywhere.